packages/angular_devkit/core/src/experimental/jobs/api.ts

```diff
@@ -131,7 +131,7 @@ export enum JobOutboundMessageKind {
   OnReady = 'c',
   Start = 's',
   End = 'e',
-  Pong = 'p',
+  Ping = 'p',
```

Comment:
😅

Reply:
I’m not sure that’s a typo

Reply:
I see, ping <-> pong

Reply:
For a network ping, ICMP uses echo request (type 8) and echo reply (type 0) for the messages.